### PR TITLE
added a 'n' shortcut/accesskey to jump to the next unread msg

### PR DESCRIPTION
**Pull request by staabm**

### CODE REVIEW

File: res/views/thread.html.twig [hunk 1]
```diff
@@ -72,14 +72,23 @@
...
                     $('.thread-navigation').remove();
                     return;
                 }
                 window.location.hash = '#' + unreadEmailIds[i];
                 i++;
+            };
+            
+            $('#next-unread').click(nextUnread);
+            $( document ).keypress(function( event ) {
+                if ( event.which == 110 ) {
```
Comment (mnapoli) [inline res/views/thread.html.twig]: Thanks, could you add a comment explaining which key it is?

**Comment by staabm** [comment]
added a comment and rebased the commit.

**Comment by mnapoli** [comment]
:+1: thanks

**Comment by mnapoli** [comment]
I don't have ssh access at the moment but I'll try to remember to deploy all that tonight

**Comment by staabm** [comment]
thx for your fast response!

**Comment by mnapoli** [comment]
It's deployed!

**Pull request merged**